src/lib/fetch.ts

```diff
@@ -11,6 +12,16 @@ import { debug } from './logger';
 
 const SUCCESS_NO_CONTENT = 204;
 
+function getProxy() {
+  // this is something of an industry standard env var, hence the checks for different casings
+  const proxy = process.env.HTTPS_PROXY || process.env.https_proxy;
```

Comment:
TIL that env variables aren't case-insensitive

<img width="283" alt="Screen Shot 2022-11-28 at 3 37 36 PM" src="https://user-images.githubusercontent.com/33762/204404225-fbd9506e-22a5-41e5-893b-a80b824836b5.png">